README.md

```diff
@@ -1,29 +1,2 @@
-# whiteboard-course-management
-Course management web application inspired by Blackboard, made for a computer science projects course
-
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
-
-Starts a live-reloading development server.
-
-### `npm test`
-
-Launches a test runner in interactive watch mode.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder, optimizing the build for best performance.
-
-
-## Project Requirements
-
- - Three user types: Admin, Professor, and Student
- - The app is authenticated to differentiate between users and permissions
- - Administrators can create courses and assign professors and students to courses
- - Professors can post course content (lectures/resources/etc) and assignments
- - Students can view course content and grades, and can post submissions to assignments
- - The project must include automated testing
+Made with Yarn 
+Build for the Dash board muck up for the Project 
```

Comment:
replaced the useful README.md with one that seems auto-generated from some dependency